include/roboteam_ai/world/Field.h

```diff
@@ -185,6 +217,34 @@ class Field {
     // The top most point of the opponents goal (this point is on the right line).
     std::optional<Vector2> theirTopGoalSide;
 
+    // The top position, point with the highest y-coordinate, of the left penalty line (our penalty line).
+    std::optional<Vector2> leftPenaltyLineTop;
+
+    // The bottom position, point with the lowest y-coordinate, of the left penalty line (our penalty line).
+    std::optional<Vector2> leftPenaltyLineBottom;
+
+    // The top position, point with the highest y-coordinate, of the right penalty line (their penalty line).
+    std::optional<Vector2> rightPenaltyLineTop;
```

Comment:
obvious comment

Reply:
Obvious comment indeed if you look at the documentation sketch, but less obvious if you do not have this information. In the past I created games where the top position had the lowest y-coordinate which is also a design choice.